index.d.ts

```diff
@@ -5,6 +5,12 @@ Symbol event names can be used to avoid name collisions when your classes are ex
 */
 type EventName = string | symbol;
 
+/** The data provided as `eventData` when listening for `Emittery.listenerAdded` or `Emittery.listenerRemoved`. */
+interface EventListenerData {
+	listener: (eventData?: unknown) => void;
+	eventName?: string;
```

Comment:
Why is it optional? And it should be:
```suggestion
	eventName?: EventName;
```

Reply:
@sindresorhus When using `emitter.onAny`, an eventName is not specified as part of the function.

Reply:
> When using emitter.onAny, an eventName is not specified as part of the function.

That should be properly documented in the docs (TS/readme).

Reply:
@sindresorhus The docs literally say:
> The name of the event that was added or removed if `on` or `off` was used instead of `onAny` or `offAny`.